assets/css/style.css

```diff
@@ -123,6 +121,48 @@ a {
   width: 200px; /*PLACEHOLDER*/
 }
 
+
+/*Home Review Section*/
+.home-review-section {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+}
+.home-review-copy {
+  grid-column-start: 1;
+  grid-column-end: 4;
+  text-align: center;
+}
+.home-review-piece {
+  width: 125px;
+}
+.review-img {
+  width: 100px;
+  height: 100px;
+  object-fit: cover;
+}
+.stars {
+  color: #3f1a7a;
+  margin-top: 0;
+  margin-bottom: 5px;
+}
+.read-review-button {
+  font-family: 'Roboto', sans-serif;
+  color: #ffffff;
+  font-size: 14px;
+  background-color: #3f1a7a;
```

Comment:
The destination page uses a different color. Do we want to use this color on the destination pages for consistency?

Reply:
That sounds like a good idea.